ironfish/src/mining/manager.test.slow.ts

```diff
@@ -100,6 +100,7 @@ describe('Mining manager', () => {
 
     const { blockTransactions } = await nodeA.miningManager.getNewBlockTransactions(
       nodeA.chain.head.sequence + 1,
+      accountA,
```

Comment:
Is this needed?

Reply:
To get the sample miner's fee we need an account's spending key. We get the default account and check that it's not null in `miningManager.createNewBlockTemplate()` which is what calls `miningManager.getNewBlockTransactions()` so I think it makes sense to pass that account into the function.